terraform/modules/bigquery/main.tf

```diff
@@ -0,0 +1,14 @@
+# Set up BQ Dataset
+resource "google_bigquery_dataset" "bq_demo_dataset" {
+  project    = var.project
+  dataset_id = var.dataset_name
+  location   = var.dataset_location
+}
+
+resource "google_bigquery_table" "bq_demo_tables" {
```

Comment:
Please add a comment that for tables requiring additional config (e.g. clustering, etc) they need to be defined outside the loop